app/models/foreman_snapshot_management/snapshot.rb

```diff
@@ -33,7 +33,9 @@ def self.new_for_host(host)
     end
 
     def self.all_for_host(host)
-      host.compute_resource.get_snapshots(host)
+      snapshots = host.compute_resource.get_snapshots(host)
+      snapshots.delete(snapshots.last) if host.compute_resource.capable?(:backup_snapshot)
```

Comment:
Why is this? What is meant by backup_snapshot? Can we just handle this in the `get_snapshots` method of proxmox without adding another capability?